README.md

```diff
@@ -1,28 +1,121 @@
-# SubQuery - Dictionary 
+# 🤔 Introduction
```

Comment:
Use the same template as the [SDK README](https://github.com/capsule-corp-ternoa/ternoa-js/blob/dev/README.md) including:
- a title header: 🚚 Ternoa Dictionary
- a brief intro (use the introduction section)
- a table of contents

Reply:
Done